api/soap/mc_enum_api.php

```diff
@@ -290,11 +290,17 @@ function mci_enum_get_array_by_id( $p_enum_id, $p_enum_type, $p_lang ) {
 		$t_result['name'] = MantisEnum::getLocalizedLabel( $t_enum_string_value, $t_enum_localized_value, $p_enum_id );
 	} else {
 		$t_enum_array = MantisEnum::getAssocArrayIndexedByValues( $t_enum_string_value );
```

Comment:
This line doesn't seem to be used anymore.
`$t_enum_array = MantisEnum::getAssocArrayIndexedByValues( $t_enum_string_value );`

Reply:
Missed that. Fixed in ebac07b287bfc619dcc04333b7832dcf771d9eab, thanks !